.travis.yml

```diff
@@ -2,24 +2,23 @@
 ---
 sudo: false
 language: python
+dist: xenial
 python:
-  - 2.7
   - 3.5
   - 3.6
-  - 3.7-dev
-  - pypy
-  - pypy3
+  - 3.7
 install:
-  - pip install -r test-requirements.txt
-  - pip install -r requirements.txt
-script: nosetests
+  - pip install -r requires/development.txt
+script: 
+  - nosetests
+  - python setup.py build_sphinx
```

Comment:
I'm totally confused why you'd do this -- we manage the docs externally and it just eats of CPU time with no retained build artifact.